README.md

```diff
@@ -33,5 +33,6 @@ The Community Committee is an autonomous committee that collaborates alongside t
 - Myles Borins ([MylesBorins](https://github.com/MylesBorins))
 - Ashley Williams ([ashleygwilliams](https://github.com/ashleygwilliams))
 - Gregor Martynus ([gr2m](https://github.com/gr2m))
+- Emily Rose ([emilyrose](https://github.com/emilyrose))
```

Comment:
Total minor nit you can ignore: The ordering here seems a little arbitrary to me. It might be a good time to decide on an ordering a stick with it. Alphabetical-by-GitHub-handle is used in the main Node.js README, if that's useful info. But any ordering that is obvious when you look at the list will do.